src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -22,4 +22,7 @@
     @ManyToMany(cascade = CascadeType.MERGE, mappedBy = "cartsWhichContainsThisProduct")
     List<Product> productList = new ArrayList<>();
 
+    @OneToOne(mappedBy = "cart")
```

Comment:
Ja w swojej wersji mam: 
@OneToOne
    @JoinColumn(name = "ORDER_ID")
    Order order;

(informacyjnie)

Reply:
mógłbyś sprecyzować to ? Cart to nie moja klasa. Jak coś zrobiłem to tylko żeby działało i abyś to zmienił po swojej stronie.